template.json

```diff
@@ -1366,6 +1366,9 @@
                 "environmental": false,
                 "physical": false
             },
+            "ritual": {
```

Comment:
When adding template.json fields always add the appropriate types as well.

You can find these under `src/module/types`. For this it would be within `item/Spell.ts` and the types would be:
```
export interface RitualData {
  type: string
}
```
and within the `SpellPartData`
```
...
ritual: RitualData
...
```

Reply:
I added it